app/jobs/event/approval_cleanup_job.rb

```diff
@@ -0,0 +1,55 @@
+# frozen_string_literal: true
+
+#  Copyright (c) 2024, Pfadibewegung Schweiz. This file is part of
+#  hitobito_pbs and licensed under the Affero General Public License version 3
+#  or later. See the COPYING file at the top-level directory or at
+#  https://github.com/hitobito/hitobito_pbs.
+
+class Event::ApprovalCleanupJob < RecurringJob
+  run_every 1.day
+
+  def perform_internal
+    clean_approval_comments(approval_ids)
+  end
+
+  def cutoff_date = 3.months.ago
+
+  def old_event_ids
+    Event::Date
+      .select('MAX(finish_at) AS finish_at').where('finish_at < ?', cutoff_date)
```

Comment:
I begin to question if it wrong to only consider the `finish_at` and no the `start_at`. It might work that I put `MAX(finish_at, start_at)` in the `select`.

Patches/Suggestions welcome.

Reply:
I was a little more complicated, but it now looks at all dates of an event and works with start and finish-dates as the apps allows.